highwind-ai/examples
# Add machine translation example to the new repo

## Overview

This PR moves the machine-translation example from the old repo to the new repo. 

## Changes

- Move over all machine-translation files
- Update notebook to push model to the Melio Hugging Face space: https://huggingface.co/MelioAI/machine-translation
- Add instructions on how to get access tokens from Hugging Face to be able to log in to Hugging Face from the notebook.
- Add download and upload file methods from Hugging Face.
- Update the `deployment` README 
- Update `deployment` main.py script with docstrings
- Add `image_name.txt` example
- Tested the local deployment still works 
- Used the Highwind docs (To ensure steps are correct in the docs) to create and link an Asset, use case and deployment. 

## To test
- Run the machine translation notebook and check that files are updated on Hugging Face once training is done
- Test the download functionality
- Go to the `deployment` folder and use the README to locally test the deployment
- Optionally: Create an Asset and Use Case, link the asset and deploy the model.
